config/goaccess.conf

```diff
@@ -10,7 +10,7 @@
 # The following time format works with any of the
```

Comment:
Is this change needed for fuzzing?

Reply:
Yep. Moved it to the Dockerfile, though.